libnmstate/netapplier.py

```diff
@@ -160,13 +157,7 @@ def _apply_ifaces_state(
     except nm.checkpoint.NMCheckPointCreationError:
         raise NmstateConflictError("Error creating a check point")
     except NmstateError:
-        # Assume rollback occured, revert IPv6 stack state.
-        # Checkpoint rollback is async, there is a need to wait for it to
-        # finish before proceeding with other actions.
-        # TODO: https://nmstate.atlassian.net/browse/NMSTATE-103
         time.sleep(5)
```

Comment:
You left the "sleep" part, therefore the follwing is still relevant.
```
# Assume rollback occurred,
# Checkpoint rollback is async, there is a need to wait for it to
# finish before proceeding with other actions.
```

Reply:
Oops. Yes, you are right. Thanks!